Makefile

```diff
@@ -110,6 +110,13 @@ libtwin.a_files-y += backend/fbdev.c
 libtwin.a_files-y += backend/linux_input.c
 endif
 
+ifeq ($(CONFIG_BACKEND_VNC), y)
+BACKEND = vnc
+libtwin.a_files-y += backend/vnc.c
+libtwin.a_cflags-y += $(shell pkg-config --cflags neatvnc aml pixman-1)
```

Comment:
In #55, it also uses pixman features. Therefore, regardless of who merges first, another person needs to pay attention to the adjustment of pixman patch in the Makefile.